config/engines.yml

```diff
@@ -141,6 +141,14 @@ phpmd:
     - "**.php"
     - "**.module"
     - "**.inc"
+radon:
+  image: codeclimate/codeclimate-radon
+  description: Python tool used to compute Cyclomatic Complexity.
+  community: true
+  enable_regexps:
```

Comment:
@wfleming Good question. I included the `enable_regexps` because that seemed to be the pattern present for the other community engines, like gofmt.

However, I personally think we'll auto-enable certain community engines eventually and I'd advocate that.

That seems like the way we'd offer the same easy first-time-analysis-without-config experience to users with projects in languages like `python`, and `go`